src/matlab2tikz.m

```diff
@@ -1911,6 +1919,11 @@ function assertRegularAxes(handle)
             || ~abs(lineWidth-matlabDefaultLineWidth) <= m2t.tol
         lineOpts = opts_add(lineOpts, 'line width', sprintf('%.1fpt', lineWidth));
     end
+
+    % print no lines
+    if isNone(lineStyle) || lineWidth==0
```

Comment:
Nah keep it as it is. Otherwise, there is always the possibiilty that a check fails etc.